### PR TITLE
Update premailing (fixes mozilla/payments-ui#189)

### PR DESCRIPTION
**Pull request by muffinresearch**
See also: https://github.com/mozilla/payments-ui/pull/196

payments_ui

### CODE REVIEW

File: payments_service/braintree/templates/braintree/emails/subscription_header.html [hunk 1]
```diff
@@ -10,5 +10,6 @@
   <body id="email">
     <main>
       <div id="wrapper">
-        <div style="background-image: url(http://placeholder)" id="logo"></div>
+        <div id="logo"></div>
         <h1>{{ seller.name }}</h1>
+        <p class="user-email">placeholder@placeholder.com</p>
```
Comment (andymckay) [inline payments_service/braintree/templates/braintree/emails/subscription_header.html:15]: Actually looking at this, its kinda wierd, why does an email to user have their email address in the top of it? cc @brampitoyo

File: payments_service/settings/base.py [hunk 2]
```diff
@@ -110,7 +110,8 @@
 EMAIL_ALLOWED_LIST = ['.*@mozilla\.com$']
 
 # The root URL for all CSS etc.
-EMAIL_URL_ROOT = 'http://192.168.99.100:8000'
+EMAIL_URL_ROOT = (os.environ.get('EMAIL_URL_ROOT') or
+                  'http://192.168.59.103:8000')
```
Comment (andymckay) [inline payments_service/settings/base.py:114]: what ip is that?

File: payments_service/braintree/templates/braintree/emails/subscription_charged_successfully.html [hunk 3]
```diff
@@ -1,13 +1,41 @@
...
+        <tr class="cc">
+            <th scope="row">Paid with</th>
+            {# __CC_TYPE__ is post-process replaced #}
+            <td><img class="card" src="/svg/__CC_TYPE__.svg" width="40" /> ending in {{ cc_truncated_id }}</td>
+        </tr>
+        <tr class="desc">
+            <th scope="row">{{ product.description }}</th>
+            <td>${{ moz_trans.amount }} <span>per month</span></td>
+        </tr>
+        <tr class="period" colspan="2">
```
Comment (andymckay) [inline payments_service/braintree/templates/braintree/emails/subscription_charged_successfully.html]: haven't seen colspan in ages

**Comment by andymckay** [comment]
r+wc looks great, will try to file bugs on the bits we are missing (localisation etc)

File: payments_service/settings/base.py (hunk 2 above)
Comment (muffinresearch) [inline payments_service/settings/base.py:114]: Mine and the default boot2docker one.

File: payments_service/braintree/templates/braintree/emails/subscription_charged_successfully.html (hunk 3 above)
Comment (muffinresearch) [inline payments_service/braintree/templates/braintree/emails/subscription_charged_successfully.html]: Actually need to remove that I think. I was trying to collapse things to better meet the mock which doesn't have 'Period' at all.

File: payments_service/settings/base.py (hunk 2 above)
Comment (andymckay) [inline payments_service/settings/base.py:114]: Are you using docker-machine, it defaults to http://192.168.99.100 for the first machine now. But all of these will change, so having the `os.environ` there is great.

File: payments_service/braintree/templates/braintree/emails/subscription_charged_successfully.html [hunk 4]
```diff
@@ -1,13 +1,41 @@
...
+            <td>{{ date }}</td>
+        </tr>
+        <tr>
+            <th scope="row">Receipt #</th>
+            <td>{{ transaction.uuid }}</td>
+        </tr>
+        <tr class="cc">
+            <th scope="row">Paid with</th>
+            {# __CC_TYPE__ is post-process replaced #}
+            <td><img class="card" src="/svg/__CC_TYPE__.svg" width="40" /> ending in {{ cc_truncated_id }}</td>
```
Comment (muffinresearch) [inline payments_service/braintree/templates/braintree/emails/subscription_charged_successfully.html]: Oops need to do an alt attr here I think.

File: payments_service/settings/base.py (hunk 2 above)
Comment (andymckay) [inline payments_service/settings/base.py:114]: Could we use http://pay.dev there?

Comment (muffinresearch) [inline payments_service/settings/base.py:114]: No, I'm not using that yet.

**Pull request merged**